presentations/en/2019-07-xx-Mandate.md

```diff
@@ -1,47 +1,128 @@
-# IT Strategy
+# Mandate
+
+## IT Strategy
+
+2019-07-xx
+
+---
 
 ## Vision
 
-### Future Testimonials
+### Image of the future
+
+The [Canadian Experience](https://github.com/sara-sabr/ITStrategy/blob/master/Strategies/HumanDevelopmentLifeCycle.md) is when Canadians are benefiting seamlessly of the federal government services through modern technologies.
+
+--
+
+### Testimonials
 
 I just lost my job. It's been a month since my last paycheque and today I received a deposit of $1245.28 in my TD checking account. With this, I was able to buy groceries for my family while job hunting full time.
 
 As a student who isn't taking any courses this summer, I decided to look for work. My Government of Canada LinkedIn friend just sent me the perfect job opportunity to be an intern at Shopify. I'm excited for my interview next week!
 
 ---
 
+## Mission
```

Comment:
change to Mission (reason for existence)
and remove sub-section "Team's reason for existence"

Reply:
I changed the `(...)` to a subtitle because it's since this is a presentation.  I think it's a good idea to remove mention of "team's", I removed it from other subtitles.

We could keep the `(...)` if we want.